content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -2,40 +2,122 @@
 Cycle counts
 ============
 
-In most companies, the stock is only counted once a year. That's why by default, after making an
-inventory adjustment in Odoo, the scheduled date for the next count is set on the 31st of
-December. However, for some businesses it's crucial to have an accurate inventory count at all
-times.
+For most companies, warehouse stock only needs to be counted once a year. This is why, by default,
+after making an *inventory adjustment* in Odoo, the scheduled date for the next inventory count is
+set for the 31st of December of the current year.
 
-The goal of cycle counts is to keep critical stock levels accurate by counting more often at key
-locations.
+However, for some businesses, it's crucial to have an accurate inventory count at all times. These
+companies use *cycle counts* to keep critical stock levels accurate. Cycle counting is a method by
+which companies count their inventory more often in certain *locations*, to ensure that their
+physical inventory counts match their inventory records in Odoo.
 
-Configuration
-=============
+Activate storage locations
+==========================
 
-In Odoo, cycle counts are location-based. The frequency of the counts is defined by the storage
-location. To activate storage locations, go to :menuselection:`Inventory --> Configuration -->
-Settings --> Warehouse` and activate the :guilabel:`Storage Locations` setting. Next, click
-:guilabel:`Save` to apply the setting.
+In Odoo, cycle counts are location-based. Therefore, the *storage locations* feature needs to be
+enabled before performing a cycle count.
 
-.. image:: cycle_counts/storage-locations-setting.png
+To enable this feature, navigate to :menuselection:`Inventory app --> Configuration --> Settings`,
+and scroll down to the :guilabel:`Warehouse` section. Then, click the checkbox next to
+:guilabel:`Storage Locations`. Click :guilabel:`Save` to save all changes.
+
+.. image:: cycle_counts/cycle-counts-enabled-setting.png
    :align: center
-   :alt: Enable Storage Locations in Odoo settings.
+   :alt: Enabled storage locations setting in inventory settings.
+
+Change inventory count frequency by location
+============================================
 
-Change the inventory frequency
-==============================
+Now that the storage locations setting is enabled, the inventory count frequency can be changed for
+specific locations created in the warheouse.
 
-To change a location's inventory frequency, first, go to the locations by clicking
-:menuselection:`Inventory --> Configuration --> Locations`.
+To view (and edit) locations, navigate to :menuselection:`Inventory app --> Configuration -->
+Locations`. This reveals a :menuselection:`Locations` page containing every location currently
+created and listed in the warehouse.
```

Comment:
```suggestion
Locations`. This reveals a :guilabel:`Locations` page containing every location currently created
and listed in the warehouse.
```
changing it to a guilabel here because the navigation to the page is already complete, and this is the title that appears on the page/screen/UI